.github/workflows/create_search_index.yml

```diff
@@ -18,6 +18,14 @@ jobs:
           ruby-version: 2.7
           working-directory: docs
           bundler-cache: true
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: yarn
+          cache-dependency-path: docs/_frontend/yarn.lock
```

Comment:
We don't need a lock file here, because you install a npm package globally.

Reply:
Yes. But the lock file will be required for the `Build frontend` step that comes later on, no ?